src/Database/Connection.php

```diff
@@ -223,7 +223,7 @@ public function connect()
     {
         try {
             return $this->_driver->connect();
-        } catch (\Exception $e) {
+        } catch (Exception $e) {
```

Comment:
Shouldn't a `use Exception` be added at top of file?

Reply:
I added it.

Reply:
Weird that the Ide didnt. Thank you.